help/destinations/catalog/email-marketing/oracle-eloqua-api.md

```diff
@@ -102,48 +113,51 @@ Read [Activate profiles and segments to streaming segment export destinations](/
-        |Source Field|Target Field| Mandatory|
+1. In the **[!UICONTROL Select target field]** window, choose **[!UICONTROL Select identity namespace]** and select an identity, or choose **[!UICONTROL Select custom attributes]** category and type in the desired attribute name in the **[!UICONTROL Attribute name]** field. The attribute name that you provide should match an existing contact attribute in [!DNL Oracle Eloqua]. See [[!DNL create a contact]](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-rest-api/op-api-rest-1.0-data-contact-post.html) for the exact attribute names that you can use in [!DNL Oracle Eloqua].
+    * Repeat these steps to add both the required and any desired attribute mappings between your XDM profile schema and [!DNL Oracle Eloqua]:
+        | Source Field | Target Field | Mandatory |
         |---|---|---|
-        |`xdm: personalEmail.address`|`Attribute: emailAddress`| Yes |
         |`IdentityMap: Eid`|`Identity: EloquaId`| Yes |
-
-    * An example using these mappings is shown below:
+        |`xdm: personalEmail.address`|`Attribute: emailAddress`| Yes |
+        |`xdm: personName.firstName`|`Attribute: firstName`| |
+        |`xdm: personName.lastName`|`Attribute: lastName`| |
+        |`xdm: workAddress.street1`|`Attribute: address1`| |
+        |`xdm: workAddress.street2`|`Attribute: address2`| |
+        |`xdm: workAddress.street3`|`Attribute: address3`| |
+        |`xdm: workAddress.postalCode`|`Attribute: postalCode`| |
+        |`xdm: workAddress.country`|`Attribute: country`| |
+        |`xdm: workAddress.city`|`Attribute: city`| |
+
+    * An example with the above mappings is shown below:
     ![Platform UI screenshot example with attribute mappings.](../../assets/catalog/email-marketing/oracle-eloqua-api/mappings.png)
 
-        >[!IMPORTANT]
-        >
-        >Both the `emailAddress` and `EloquaId` target attribute mappings are mandatory.
+>[!IMPORTANT]
+>
+>* Attributes specified in the **[!UICONTROL Target field]** should be named exactly as specified in the [[!DNL update a contact API endpoint fields]](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-rest-api/op-api-rest-1.0-data-contact-id-put.html) as these attributes will form request body.
+>* Attributes specified in the **[!UICONTROL Source field]** do not follow any such restriction. You can map it based on your need, however if the data format is not correct when pushed to [!DNL Oracle Eloqua] it will result in an error. For example, you can map the **[!UICONTROL Source field]** identity namespace `contact key`, `ABC ID` etc. to **[!UICONTROL Target field]** : `EloquaId` after ensuring that the ID values match the format that is accepted by [!DNL Oracle Eloqua].
+>* The `EloquaID` mapping is mandatory to update attributes corresponding to the Identity. 
+>* The `emailAddress` mapping is also required as without it the API throws an error as shown below:
```

Comment:
Which API throws this error? Is it shown in the Platform UI?

Let's rephrase this as: The `emailAddress` mapping is required. Without it, the API throws an error as shown below: